server/repository/src/db_diesel/requisition/requisition_row.rs

```diff
@@ -35,13 +35,20 @@ table! {
         min_months_of_stock -> Double,
         approval_status -> Nullable<crate::db_diesel::requisition::requisition_row::RequisitionRowApprovalStatusMapping>,
         linked_requisition_id -> Nullable<Text>,
-        is_sync_update -> Bool,
         program_id -> Nullable<Text>,
         period_id -> Nullable<Text>,
         order_type -> Nullable<Text>,
     }
 }
 
+table! {
```

Comment:
this like a view on the requisition table is it? It's very neat!

Reply:
I guess it's like a view, except you can edit it, it's treating requisition table as only having id and is_sync_update